Monika After Story/game/script-topics.rpy

```diff
@@ -12030,3 +12030,44 @@ label monika_perspective:
     m 1hksdlb "Well I hope that helped you understand at least a little, [player]."
     m "It's the best way I could try and explain it!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_relationcomfort",
+            category=["romance","advice"],
+            prompt="Comfortable in a relationship",
+            random=True
+        )
+    )
+
+label monika_relationcomfort:
+    m 1wud "You know [player], I've been thinking about our relationship lately..."
```

Comment:
I would put a comma after "know".

Reply:
I don't think that's right and is unncessary. -w-

Reply:
Usually, there are commas right before [player] in most situations, but I do think it's a bit more arguable when it's right near the start of the line. 

To play devil's advocate, you could read those first 3 words as Monika saying something about how _you know someone named [player]_ as opposed to referring to the player directly. 

Sure, I'll let it pass. See what other people think about it.